.github/workflows/backport-pr.yml

```diff
@@ -0,0 +1,41 @@
+name: Backport PR
+on:
+  pull_request:
+    types: [ labeled, closed ]
+
+permissions: {}
+
+jobs:
+  backport:
+    name: Backport PR
+    runs-on: ubuntu-latest
+    if: >
+      github.event.pull_request.merged
+      && (
+        github.event.action == 'closed'
+        || (
+          github.event.action == 'labeled'
+          && contains(github.event.label.name, 'backport')
```

Comment:
Is the "PR merged && closed" condition, without the `backport` label, a "regular" PR? Would a backport PR be created if there isn't an active release branch where it's needed?

Reply:
ah yeah, it should only run when the pr is merged and labeled correctly

Reply:
In that case, should the comparison operator be `&&` then, not `||`?

Reply:
Yep, I updated it to include two different scenarios:
```
      github.event.pull_request.merged
      && (
        ( github.event.action == 'closed' && contains(github.event.label.name, 'backport') )
        || 
        ( github.event.action == 'labeled' && contains(github.event.label.name, 'backport') )
      )
```
The two different statements for `github.event.action` are needed to run the workflow when the PR is labeled before vs. after merging.